changelog/july2023/2023-07-26-containers-added-triggers-for-serverless-containers.mdx

```diff
@@ -0,0 +1,13 @@
+---
+title: Triggers for Serverless Containers in Beta
+status: added
+author:
+    fullname: 'Join the #[xxx] channel on Slack.'
+    url: 'https://slack.scaleway.com'
+date: 2023-07-26
+category: serverless
+product: containers
+---
+
+Serverless Containers can now be triggered from Message Queues hosted in Messaging and Queuing SQS namespaces. Simply add an SQS Trigger from the console and when a new message is sent to the queue, it will trigger your Containers on the / path.
```

Comment:
```suggestion
Serverless Containers can now be triggered from Message Queues hosted in Messaging and Queuing SQS namespaces. Simply add an SQS Trigger from the console and when a new message is sent to the queue, it will trigger your Containers on the `/` path.
```